cmd/osbuild-image-tests/main_test.go

```diff
@@ -395,16 +395,16 @@ func getAllCases() ([]string, error) {
 
 // runTests opens, parses and runs all the specified testcases
 func runTests(t *testing.T, cases []string) {
-	for _, path := range cases {
-		t.Run(path, func(t *testing.T) {
-			f, err := os.Open(path)
+	for _, p := range cases {
```

Comment:
Are there no users of this in the tests that need to be adjusted?

Reply:
IIRC I don't think so.